tests/Util/ClassSourceManipulatorTest.php

```diff
@@ -643,6 +681,19 @@ public function getAddOneToOneRelationTests(): \Generator
                 isNullable: true,
             ),
         ];
+
+        yield 'one_to_one_cascade_persist_only' => [
+            'User_simple.php',
+            'User_simple_owning.php', // Same because ['persist', 'remove'] is forced
+            new RelationOneToOne(
+                propertyName: 'userProfile',
+                targetClassName: \App\Entity\UserProfile::class,
+                targetPropertyName: 'user',
+                isOwning: true,
+                isNullable: true,
+                cascade: ['persist'],
+            ),
+        ];
```

Comment:
Let's remove the many_to_many and one_to_one test cases. It's enough to make sure that a "singular" and "collection" relationship are tested.